governance-exercises/VoteWizard_poll_exercise_04/Emergency Response Poll - Risk mitigation options - August 9, 2023.md

```diff
@@ -0,0 +1,72 @@
+---
+title: Emergency Response Poll - Risk mitigation options - August 9, 2023
+summary: Signal your preferred option to mitigate the risk of a potential exploit
+discussion_link: https://forum.makerdao.com/t/...
+parameters:
+    input_format: single-choice
+    victory_conditions:
+        - { type : plurality }
+    result_display: single-vote-breakdown
+version: v2.0.0
+options:
+   0: Abstain
+   1: Fix B
+   2: Fix B and Fix A
+start_date: 2023-08-09T10:00:00
+end_date: 2023-08-09T15:00:00
+---
+# Poll: Emergency Response Poll - Risk mitigation options - August 9, 2023
+
+The Governance Facilitators have placed a Governance Poll into the voting system on behalf of a Protocol Scope Facilitator. This [Emergency Governance Poll](https://mips.makerdao.com/mips/details/MIP24) will be active for 5 hours beginning on Wednesday, August 9 at 10:00 UTC.
+
+**This is a binary vote.**
+- **You may vote for a single option.**
+- **You should vote for the option which you prefer.**
+- **If you would accept either option, you should vote 'Abstain'.**
+
+# Review
```

Comment:
Worth maybe doing some damage control here as well. Ie repeating the good news from technical teams, Collateral and dai holders not at risk, etc...